src/port_details.rs

```diff
@@ -0,0 +1,66 @@
+pub const MOST_COMMON_PORTS: &[u16] = &[
```

Comment:
where is this list coming from?

we should have a reference like https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers and in the future we can update it from the ref.

Reply:
check the commit message, its coming from nmap-services.
I will add this as a comment as well in next update.